tf2_ros/docs/Doxyfile

```diff
@@ -0,0 +1,29 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "tf2_ros"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "This package contains the ROS bindings for the tf2 library, for both Python and C++."
+
+INPUT                  = ../include ./mainpage.dox
+RECURSIVE              = YES
+OUTPUT_DIRECTORY       = doc_output
+
+EXTRACT_ALL            = YES
+SORT_MEMBER_DOCS       = NO
+
+GENERATE_LATEX         = NO
+
+ENABLE_PREPROCESSING   = YES
+MACRO_EXPANSION        = YES
+EXPAND_ONLY_PREDEF     = YES
+PREDEFINED             += TF2_PUBLIC=
+
+# Tag files that do not exist will produce a warning and cross-project linking will not work.
+TAGFILES += "../../../../doxygen_tag_files/cppreference-doxygen-web.tag.xml=http://en.cppreference.com/w/"
```

Comment:
If this is in the docs folder, shouldn't there be an extra `../` here and below?